index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+  
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="portfolio.css">
+
+</head>
+<body>
+    <!-- TOOLBAR -->
+    <header class="toolbar">
+        <div class="logo"> Welcome </div>
```

Comment:
- [ ] You have a great `header` structure and the elements inside especially the logo, I would suggested that it is placed in an `<a>` tag instead of a `<div>` tag.  This is important because usually a logo is clickable and at most times a link.

Reply:
Done, thank you